tests/integration_tests/build/test_coverage.py

```diff
@@ -20,7 +19,7 @@
 if utils.is_io_uring_supported():
     COVERAGE_DICT = {"Intel": 82.97, "AMD": 82.14, "ARM": 82.43}
 else:
-    COVERAGE_DICT = {"Intel": 80.13, "AMD": 79.28, "ARM": 79.34}
+    COVERAGE_DICT = {"Intel": 80.47, "AMD": 79.28, "ARM": 79.34}
```

Comment:
I can't test other platform, because I don't have other platform environment.
Maybe the test fails with a coverage error.